### ISSUE TITLE
Add a hex color wheel to the menu bar

### ISSUE THREAD

**Issue by SethWalkeroo** [SethWalkeroo/Quiet-Text#2]
Right now you can edit the colors of the application in the settings.json file, but it would be nice if you could also select and look for hex colors inside the text editor. Implement a color wheel of sorts that allows users to copy and paste hex values without having to use google.

**Comment by Ryuno-Ki** [comment]
I'd like to take a stab on it.

I already discovered tkinter's colorchooser, which should work here.
Can you assign it to me?

**Comment by SethWalkeroo** [comment]
@Ryuno-Ki Sure, go ahead!

**Comment by SethWalkeroo** [comment]
@Ryuno-Ki Let me know if you have any questions about the existing code. I haven't taken the time to go through and comment everything so I am aware that it might be difficult to read.

**Comment by Ryuno-Ki** [comment]
I filled #3.

Some bits required research (since Python's docs on `tkinter.choosecolor.askcolor()` are … sparse), but it works for the most parts.

I tested it locally on my Sabayon Linux. Sadly, the Python interpreters here are compiled without Tk support, so I wrote a Dockerfile to run the code in.

```
FROM ubuntu
RUN apt update && apt install -y python3-tk x11-apps
ENV DISPLAY :0
RUN mkdir /code
WORKDIR /code
ADD . /code
CMD ["/usr/bin/python3", "-m", "quiet"]
```

Once I prepared my environment like this:
`XAUTH="$HOME/.Xauthority"`

and built an image using
`docker build -t quiet .`

I can run it with
`docker run --network=host --rm -e DISPLAY=$DISPLAY -v $XAUTH:/root/.Xauthority quiet` then (full access to host environment, so the executed code should be better trusted).

Shall I add that to `README.md` or `CONTRIBUTING.md`?
